common-lib/src/cmr/common/concepts.clj

```diff
@@ -1,10 +1,34 @@
 (ns cmr.common.concepts
   "This contains utility functions and vars related to concepts in the CMR"
-  (:require [clojure.set]
+  (:require [clojure.set :as cset]
             [clojure.string :as str]
             [cmr.common.util :as util]
             [cmr.common.services.errors :as errors]))
 
+(def generic-concept-prefix->concept-type
+  "Maps a generic concept id prefix to the concept type."
+  {"X" :generic
+   ;; TODO: Generic work: Can I read a non common config file to read in the generics instead of coding them? This is primarily used for validation.
+   "DQS" :dataqualitysummary
+   "OO" :orderoption
+   "SO" :serviceoption
+   "SE" :serviceentry
+   "G" :grid})
+
+(def generic-concept-types->concept-prefix
+  "Gets an array of generic concept types."
+  (cset/map-invert generic-concept-prefix->concept-type))
+
+(defn get-generic-concept-types-array
+  "Gets the array of generic concept types."
+  []
+  (into [] (keys generic-concept-types->concept-prefix)))
```

Comment:
can be just (vec (keys generic-concept-types->concept-prefix)) rather than (into [] (keys generic-concept-types->concept-prefix))

Reply:
Yes, thanks.